src/main/java/com/redhat/rhjmc/containerjfr/commands/Command.java

```diff
@@ -71,6 +71,6 @@ default boolean validateTargetId(String targetId) {
     }
 
     default boolean validateRecordingName(String name) {
-        return name.matches("[\\w-_]+(\\.jfr)?");
+        return name.matches("[\\w-_]+(\\.\\d)?(\\.jfr)?");
```

Comment:
What happens when this gets to double-digit rename suffixes?

Reply:
My bad, for some reason I had remembered wrong and thought it only renamed up to 9, it's fixed now.